x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -498,7 +498,44 @@ func kibanaFetchPolicy(cfg setupConfig, client *kibana.Client, streams *cli.IOSt
 	if err != nil {
 		return nil, err
 	}
-	return findPolicy(cfg, policies.Items)
+	packagePolicies, err := kibanaFetchPackagePolicies(cfg, client, streams)
+	if err != nil {
+		return nil, err
+	}
+	return findPolicy(cfg, policies.Items, packagePolicies)
```

Comment:
Looking at the code we use the order from the slice to select which policy to fallback, Is that idempotent?

Reply:
idempotent in which way? my understanding of idempotent is all inputs act as fixed points on the method, which this does not.